src/commands/readme.rs

```diff
@@ -203,3 +214,25 @@ fn find_readme_template<'a>(
         None
     })
 }
+
+fn fix_doc_links(pkg_name: &str, readme: &str, doc_uri: Option<&str>) -> String {
```

Comment:
I don't think it's urgent but it'd be great to add some more documentation here - at a first glance it's not obvious what this functions is supposed to do - either at the function level or at the first two arms of the `match`

Reply:
Opened another PR w/ more readable code, fixes & tests: https://github.com/paritytech/cargo-unleash/pull/19